src/next-transpile-modules.js

```diff
@@ -226,8 +226,12 @@ const withTmInitializer = (modules = [], options = {}) => {
           );
 
           if (nextGlobalCssLoader) {
-            nextGlobalCssLoader.issuer = { or: [matcher, nextGlobalCssLoader.issuer] };
-            nextGlobalCssLoader.include = { or: [...modulesPaths, nextGlobalCssLoader.include] };
+            nextGlobalCssLoader.issuer = {
+              or: nextGlobalCssLoader.issuer ? [matcher, nextGlobalCssLoader.issuer] : [matcher],
+            };
+            nextGlobalCssLoader.include = {
+              or: nextGlobalCssLoader.include ? [...modulesPaths, nextGlobalCssLoader.include] : modulesPaths,
```

Comment:
So what does it mean if you need to import global CSS from an imported transpiled module? Does it just not work? Or does it work natively?